test/integration/aggregates/aggregates_integration_test.go

```diff
@@ -24,6 +24,7 @@ package aggregates
 
 import (
 	"fmt"
+	"github.com/v3io/v3io-tsdb/pkg/appender"
```

Comment:
Usually impi freaks out about this.

Reply:
we don't run impi on tests but i'll move it anyway

Reply:
I thought so, but in the makefile it looks like it runs on `./...`. Oh well.

Reply:
Not a fan of impi anyway.